<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>610</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>FILE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>610</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>OPEN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>650</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MyGLWidget" name="sceneWidget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="recordScreencastButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>610</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>GIF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseColorBackgroundButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>20</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>BACKGROUND COLOR</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>EDGES</string>
    </property>
    <widget class="QPushButton" name="chooseColorEdgeButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>COLOR</string>
     </property>
    </widget>
    <widget class="QComboBox" name="lineComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="sizeEdgeSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sizeEdgeSpinbox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>VERTICES</string>
    </property>
    <widget class="QPushButton" name="chooseColorVertexButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>COLOR</string>
     </property>
    </widget>
    <widget class="QComboBox" name="verticesComboBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="sizeVertexSpinbox">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>70</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sizeVertexSlider">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>300</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>PROJECTION</string>
    </property>
    <widget class="QComboBox" name="projectionComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>380</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>SCALE</string>
    </property>
    <widget class="QSlider" name="sliderIncrease">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxIncrease">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>460</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>ROTATE</string>
    </property>
    <widget class="QDoubleSpinBox" name="spinBoxX">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>68</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sliderX">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>35</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxY">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>68</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sliderY">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>66</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxZ">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>68</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sliderZ">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>96</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>311</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>MOVE</string>
    </property>
    <widget class="QDoubleSpinBox" name="spinBoxMoveX">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>68</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMoveX">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>35</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>21</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxMoveY">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>68</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMoveY">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>66</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spinBoxMoveZ">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>90</y>
       <width>68</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <double>-360.000000000000000</double>
     </property>
     <property name="maximum">
      <double>360.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="sliderMoveZ">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>95</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-360</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>21</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="recordImageButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>610</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SCREENSHOT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QWidget</extends>
   <header>myglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
